atmo/templates/atmo/jobs/mails/failed_run_alert_body.txt

```diff
@@ -3,7 +3,7 @@ at approximately {{ alert.run.created_at }} UTC.
 
 You may want to check the logs to see what failed in the Spark job.
 
-{% if alert.reason %}The reason for the failure that AWS reported was: {{ alert.reason }}{% endif %}
+{% if alert.reason_message %}The reason for the failure that AWS reported was: {{ alert.reason_message }} {% if alert.reason_code %}({{ alert.reason_code }}){% endif %}{% endif %}
```

Comment:
Nah, they are strings [like this](https://github.com/mozilla/telemetry-analysis-service/blob/0e9985340cbfdc5ed565f1b96da9970779be8bf4/atmo/clusters/models.py#L58-L64), e.g. `USER_REQUEST`